modules/openapi-generator-core/src/main/java/org/openapitools/codegen/meta/Stability.java

```diff
@@ -0,0 +1,37 @@
+package org.openapitools.codegen.meta;
+
+/**
+ * Represents the "stability index" of a generator or feature, based on the stability indexes defined in the node.js ecosystem.
+ */
+public enum Stability {
```

Comment:
NOTE: We could apply `Stability` to other things as well, such as the templating engine adapters.